src/Lynx/List.purs

```diff
@@ -61,6 +65,7 @@ type View
 type Column
   = { name :: Lynx.Expr.Expr
     , value :: Lynx.Expr.Key
+    , width :: Maybe Size
```

Comment:
Can we convert from `Maybe Size` to more domain-specific terminology? I think we decided on: auto, small, medium, and large as the different widths. It would be nice to speak the same language as the spec when we're working within the domain. It's fine for the JSON side of things to remain with primitives like `Maybe _`, though.